### DISCUSSION TITLE
How to unblock mailu email account from SMTP error 450 4.2.1 after sending too many emails too fast?

### DISCUSSION THREAD

**Discussion by kkarsten62** [Mailu/Mailu#3373] [User support] [answered]
This morning there was a cross-site attack on my server targeting my webpage (not to mailu). But my server was hard enough to resist to this stupid attack. But a php script sent via a mailu email account to much error log emails. This sending email account is now blocked for sending emails. The account can receive emails.

I can login in Roundcube Webmail to this email account. But when sending an email in Roundcube, in the lower right corner a red box appears for some seconds, saying:

`SMTP Fehler (450): Der Empfänger "mailto@domain.org" konnte nicht gesetzt werden (4.2.1 <mailfrom@domain.org>: SASL login name rejected: You are sending too many emails too fast.).`

In log file I see the following infos:
```
webmail_1    | errors: <ti2hjsu7> PHP Error: Invalid response code received from server (POST /?_task=mail&_unlock=loading1723488052054&_framed=1&_lang=en&_action=send)
webmail_1    | errors: <ti2hjsu7> SMTP Error: Failed to add recipient 'sendto@domain.org'. 4.2.1 <sendfrom@domain.org>: SASL login name rejected: You are sending too many emails too fast. (Code: 450) in /var/www/html/program/lib/Roundcube/rcube.php on line 1774 (POST /?_task=mail&_unlock=loading1723488511014&_framed=1&_lang=en&_action=send)
```
How can I unblock this email account (mailfrom)?

Thanks for a helping hand.

Karl ...

**Comment by nextgens** [comment, marked as answer]
MESSAGE_RATELIMIT and MESSAGE_RATELIMIT_EXEMPTION are the settings

https://mailu.io/master/configuration.html#mail-settings

If I were you I would just wait 24h.

**Reply by kkarsten62** [reply]
Thanks. It helps.
